app/assets/stylesheets/_category.scss

```diff
@@ -24,3 +24,18 @@
   color: #bfa24b;
   text-decoration: none;
 }
+
+#menu .floaters{
```

Comment:
Avoid using id selectors<br>Opening curly brace `{` should be preceded by one space